Tests/scripts/validate.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-export DEMISTO_README_VALIDATION=yes
+echo "CIRCLE_BRANCH: $CIRCLE_BRANCH CHECK_BACKWARD: $CHECK_BACKWARD CI: $CI DEMISTO_README_VALIDATION: $DEMISTO_README_VALIDATION"
```

Comment:
Note that I remove this export as it was not the reason for failure. I've opened an issue at: https://github.com/demisto/etc/issues/26612